auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -99,6 +110,13 @@ public String getEmail() {
         return mEmail;
     }
 
+    /**
+     * @return true if the user was created in the sign in flow, false if it already existed.
+     */
+    public boolean isNewEmailUser() {
```

Comment:
I learned something new today! I thought `fetchProvidersForEmail` only worked for email/password accounts.

I agree, since this is going to be much more complicated, I'll save it for another PR.